README.md

```diff
@@ -215,7 +215,7 @@ An alternative approach for definitions of third-party libraries are [triple-sla
 import UntypedJsLib from "untyped_js_lib"
 ```
 
-**Tip:** If you _must_ use `files`, enable `--files` flags or set `TS_NODE_FILES=true`.
+**Tip:** If you _must_ use `files` or `include`/`exclude`, enable `--files` flags or set `TS_NODE_FILES=true`.
```

Comment:
Nit: how about this phrasing, since some people combine "files" with "include"?

> If you _must_ use `files`, `include`, or `exclude`, enable ...

Reply:
Thanks, done. Also reworded the phrase above